### PR TITLE
Getting started links

### PR DESCRIPTION
**Pull request by zwalchuk**
Update getting started links for Odroid C1 and XU4

Connects to #44

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Change log
 
 * Update the resin-yocto-scripts submodule to 6f7a9ab326bb822196e3e48b08ef1d73d08caec6 (on master branch) [Florin]
 * Use the new offsets for u-boot env and Trusted Zone binary blob that our new u-boot (version 2017.09) expects [Florin]
+* Update getting starting links [Zach]
```
Comment (floion) [inline CHANGELOG.md]: @zwalchuk this needs to sit on top; new additions to changelog are added on top

File: odroid-c1.coffee [hunk 2]
```diff
@@ -11,9 +11,9 @@ module.exports =
 
 	instructions: commonImg.instructions
 	gettingStartedLink:
-		windows: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-C1.md#windows'
-		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-C1.md#on-mac-and-linux'
-		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-C1.md#on-mac-and-linux'
+		windows: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
+		osx: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
+		linux: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
```
Comment (floion) [inline odroid-c1.coffee]: Commit log should be instead like:

> odroid-c1.coffee: Update getting started links

> Make sure getting started links match the appropriate getting started guides.

File: odroid-xu4.coffee [hunk 3]
```diff
@@ -11,9 +11,9 @@ module.exports =
 
 	instructions: commonImg.instructions
 	gettingStartedLink:
-		windows: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-XU4.md#windows'
-		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-XU4.md#on-mac-and-linux'
-		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-ODROID-XU4.md#on-mac-and-linux'
+		windows: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
+		osx: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
+		linux: 'https://docs.resin.io/installing/gettingStarted-Odroid-C1/#creating-your-first-application'
```
Comment (floion) [inline odroid-xu4.coffee]: The commit log needs to be like the following:

> odroid-xu4.coffee: Update XU4 getting started links

File: CHANGELOG.md (hunk 1 above)
Comment (floion) [reply CHANGELOG.md]: Also, please change the commit log to be like:

> CHANGELOG: Update getting started links

**Comment by floion** [comment]
This PR also needs rebasing

File: odroid-xu4.coffee (hunk 3 above)
Comment (zwalchuk) [reply odroid-xu4.coffee]: Fixed

File: odroid-c1.coffee (hunk 2 above)
Comment (zwalchuk) [reply odroid-c1.coffee]: Fixed

**Comment by floion** [comment]
@resin-jenkins add to whitelist

**Comment by floion** [comment]
@zwalchuk can you rebase this? There seems to be an issue with how you rebased last time

**Comment by zwalchuk** [comment]
@floion These links are all going to be changing again today or early next week, I'm going to close this PR and start over doing it correctly when the new links are live

**Pull request closed without merging**